extensions/bson-jackson/build.gradle

```diff
@@ -8,14 +8,17 @@ dependencies {
     api project(':engine-processor')
     api 'de.undercouch:bson4jackson:2.15.1'
```

Comment:
We should follow up in a separate commit or PR to migrate the deps that weren't centralized, too.